src/__test__/TimelineContainer.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
only test case i see missing is if the requestHandler returns legit data, you test to ensure the TimelineContainer render expected content...i.e. a TweetContainer